tests/Doctrine/Tests/Common/Annotations/AnnotationTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Doctrine\Tests\Common\Annotations;
+
+use Doctrine\Common\Annotations\Annotation;
+use PHPUnit\Framework\TestCase;
+
+final class AnnotationTest extends TestCase
+{
+    public function testMagicGetThrowsBadMethodCallException()
+    {
+        $name = 'foo';
+
+        $annotation = new Annotation([]);
+
+        $this->expectException(\BadMethodCallException::class);
+        $this->expectExceptionMessage(sprintf(
+            "Unknown property '%s' on annotation '%s'.",
+            $name,
+            Annotation::class
```

Comment:
Shall we pull these tests into an `AbstractTestCase` for annotations instead?

Reply:
No, testing here is sufficient